Packs/PingIdentity/Integrations/PingOne/README.md

```diff
@@ -0,0 +1,412 @@
+Integrates with the PingOne Management API to unlock, create, delete and update users.  
+This integration was integrated and tested with version xx of PingOne
```

Comment:
Please change the `xx` to the PingOne version. If you don't know that version please remove this sentence.

Reply:
My apologies the first commit did not include all changes, not sure what exactly happen.  The version sentence was removed because PingOne is a SaaS application which does not have any public version.